jaeger-core/src/main/java/com/uber/jaeger/metrics/Metrics.java

```diff
@@ -95,147 +95,110 @@ public static Metrics fromStatsReporter(StatsReporter reporter) {
   }
 
   @Metric(
-      name = "traces",
-      tags = {@Tag(key = "state", value = "started"), @Tag(key = "sampled", value = "y")}
+        name = "traces",
+        tags = {
+              @Tag(key = "state", value = "started"),
+              @Tag(key = "sampled", value = "y"),
+        }
   )
   // Number of traces started by this tracer as sampled
   public Counter traceStartedSampled;
 
   @Metric(
-      name = "traces",
-      tags = {@Tag(key = "state", value = "started"), @Tag(key = "sampled", value = "n")}
+        name = "traces",
+        tags = {
+              @Tag(key = "state", value = "started"),
+              @Tag(key = "sampled", value = "n"),
+        }
   )
   // Number of traces started by this tracer as not sampled
   public Counter traceStartedNotSampled;
 
   @Metric(
-      name = "traces",
-      tags = {@Tag(key = "state", value = "joined"), @Tag(key = "sampled", value = "y")}
+        name = "traces",
+        tags = {
+              @Tag(key = "state", value = "joined"),
+              @Tag(key = "sampled", value = "y"),
+        }
   )
   // Number of externally started sampled traces this tracer joined
   public Counter tracesJoinedSampled;
 
   @Metric(
-      name = "traces",
-      tags = {@Tag(key = "state", value = "joined"), @Tag(key = "sampled", value = "n")}
+        name = "traces",
+        tags = {
+              @Tag(key = "state", value = "joined"),
+              @Tag(key = "sampled", value = "n"),
+        }
   )
   // Number of externally started not-sampled traces this tracer joined
   public Counter tracesJoinedNotSampled;
 
-  @Metric(
-      name = "spans",
-      tags = {@Tag(key = "state", value = "started"), @Tag(key = "group", value = "lifecycle")}
-  )
+  @Metric(name = "started_spans", tags = @Tag(key = "sampled", value = "y"))
   // Number of sampled spans started by this tracer
   public Counter spansStarted;
```

Comment:
can we rename this to spansStartedSampled?

Reply:
Done